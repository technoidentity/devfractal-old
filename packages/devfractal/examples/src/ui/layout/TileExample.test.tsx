import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Tile } from './TileExample'

test('Tile Example', () => {
  const component: ReactTestRenderer = renderer.create(<Tile />)
  expect(component.toJSON()).toMatchSnapshot()
})
