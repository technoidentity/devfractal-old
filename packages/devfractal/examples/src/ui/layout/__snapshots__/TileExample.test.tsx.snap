// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Example
    </h1>
    <div
      className="tile is-ancestor"
    >
      <div
        className="tile is-8 is-vertical"
      >
        <div
          className="tile"
        >
          <div
            className="tile is-parent is-vertical"
          >
            <div
              className="tile is-child is-primary notification"
            >
              <h1
                className="title"
              >
                Vertical...
              </h1>
              <h1
                className="subtitle"
              >
                Top tile
              </h1>
            </div>
            <div
              className="tile is-child is-warning notification"
            >
              <h1
                className="title"
              >
                ...tiles
              </h1>
              <h1
                className="subtitle"
              >
                Bottom tile
              </h1>
            </div>
          </div>
          <div
            className="tile is-parent"
          >
            <div
              className="tile is-child is-info notification"
            >
              <h1
                className="title"
              >
                Middle tile
              </h1>
              <h1
                className="subtitle"
              >
                With an image
              </h1>
              <figure
                className="image is-5by3"
              >
                <img
                  className=""
                  src="https://bulma.io/images/placeholders/640x480.png"
                />
              </figure>
            </div>
          </div>
        </div>
        <div
          className="tile is-parent"
        >
          <div
            className="tile is-child is-danger notification"
          >
            <h1
              className="title"
            >
              Wide tile
            </h1>
            <h1
              className="subtitle"
            >
              Aligned with the right tile
            </h1>
            <div
              className="content"
            >
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, eu pellentesque tortor vestibulum ut. Maecenas non massa sem. Etiam finibus odio quis feugiat facilisis.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="tile is-parent"
      >
        <div
          className="tile is-child is-success notification"
        >
          <div
            className="content"
          >
            <h1
              className="title"
            >
              Tall tile
            </h1>
            <h1
              className="subtitle"
            >
              With even more content
            </h1>
            <div
              className="content"
            >
              <p>
                semper diam at erat pulvinar, at pulvinar felis blandit. Vestibulum volutpat tellus diam, consequat gravida libero rhoncus ut. Morbi maximus, leo sit amet vehicula eleifend, nunc dui porta orci, quis Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam semper odio felis ut quam.
              </p>
              <p>
                Suspendisse varius ligula in molestie lacinia. Maecenas varius eget ligula a sagittis. Pellentesque interdum, nisl nec interdum maximus, augue diam porttitor lorem, et sollicitudin felis neque sit amet erat. Maecenas imperdiet felis nisi, fringilla luctus felis hendrerit sit amet. Aenean vitae gravida diam, finibus dignissim turpis. Sed eget varius ligula, at volutpat tortor.
              </p>
              <p>
                Integer sollicitudin, tortor a mattis commodo, velit urna rhoncus erat, vitae congue lectus dolor consequat libero. Donec leo ligula, maximus et pellentesque sed, gravida a metus. Cras ullamcorper a nunc ac porta. Aliquam ut aliquet lacus, quis faucibus libero. Quisque non semper leo.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Modifiers
    </h1>
    <div
      className="tile is-ancestor"
    >
      <div
        className="tile is-parent is-4 is-vertical"
      >
        <div
          className="tile is-child box"
        >
          <h1
            className="title"
          >
            One
          </h1>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, eu pellentesque tortor vestibulum ut. Maecenas non massa sem. Etiam finibus odio quis feugiat facilisis.
          </p>
        </div>
        <div
          className="tile is-child box"
        >
          <h1
            className="title"
          >
            Two
          </h1>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, eu pellentesque tortor vestibulum ut. Maecenas non massa sem. Etiam finibus odio quis feugiat facilisis.
          </p>
        </div>
      </div>
      <div
        className="tile is-parent"
      >
        <div
          className="tile is-child box"
        >
          <h1
            className="title"
          >
            Three
          </h1>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam semper diam at erat pulvinar, at pulvinar felis blandit. Vestibulum volutpat tellus diam, consequat gravida libero rhoncus ut. Morbi maximus, leo sit amet vehicula eleifend, nunc dui porta orci, quis semper odio felis ut quam.
          </p>
          <p>
            Suspendisse varius ligula in molestie lacinia. Maecenas varius eget ligula a sagittis. Pellentesque interdum, nisl nec interdum maximus, augue diam porttitor lorem, et sollicitudin felis neque sit amet erat. Maecenas imperdiet felis nisi, fringilla luctus felis hendrerit sit amet. Aenean vitae gravida diam, finibus dignissim turpis. Sed eget varius ligula, at volutpat tortor.
          </p>
          <p>
            Integer sollicitudin, tortor a mattis commodo, velit urna rhoncus erat, vitae congue lectus dolor consequat libero. Donec leo ligula, maximus et pellentesque sed, gravida a metus. Cras ullamcorper a nunc ac porta. Aliquam ut aliquet lacus, quis faucibus libero. Quisque non semper leo.
          </p>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Nesting requirements
    </h1>
    <div
      className="tile is-ancestor"
    >
      <div
        className="tile is-8 is-vertical"
      >
        <div
          className="tile"
        >
          <div
            className="tile is-parent is-vertical"
          >
            <div
              className="tile is-child box"
            >
              <h1
                className="title"
              >
                Vertical tiles
              </h1>
              <h1
                className="subtitle"
              >
                Top box
              </h1>
            </div>
            <div
              className="tile is-child box"
            >
              <h1
                className="title"
              >
                Vertical tiles
              </h1>
              <h1
                className="subtitle"
              >
                Bottom box
              </h1>
            </div>
          </div>
          <div
            className="tile is-parent"
          >
            <div
              className="tile is-child box"
            >
              <h1
                className="title"
              >
                Middle box
              </h1>
              <h1
                className="subtitle"
              >
                With an image
              </h1>
              <figure
                className="image is-5by3"
              >
                <img
                  className=""
                  src="https://bulma.io/images/placeholders/640x480.png"
                />
              </figure>
            </div>
          </div>
        </div>
        <div
          className="tile is-parent"
        >
          <div
            className="tile is-child box"
          >
            <h1
              className="title"
            >
              Wide Column
            </h1>
            <h1
              className="subtitle"
            >
              Aligned with the right Column
            </h1>
            <div
              className="content"
            >
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, eu pellentesque tortor vestibulum ut. Maecenas non massa sem. Etiam finibus odio quis feugiat facilisis.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="tile is-parent"
      >
        <div
          className="tile is-child box"
        >
          <div
            className="content"
          >
            <h1
              className="title"
            >
              Tall Column
            </h1>
            <h1
              className="subtitle"
            >
              With even more content
            </h1>
            <div
              className="content"
            >
              <p>
                semper diam at erat pulvinar, at pulvinar felis blandit. Vestibulum volutpat tellus diam, consequat gravida libero rhoncus ut. Morbi maximus, leo sit amet vehicula eleifend, nunc dui porta orci, quis Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam semper odio felis ut quam.
              </p>
              <p>
                Suspendisse varius ligula in molestie lacinia. Maecenas varius eget ligula a sagittis. Pellentesque interdum, nisl nec interdum maximus, augue diam porttitor lorem, et sollicitudin felis neque sit amet erat. Maecenas imperdiet felis nisi, fringilla luctus felis hendrerit sit amet. Aenean vitae gravida diam, finibus dignissim turpis. Sed eget varius ligula, at volutpat tortor.
              </p>
              <p>
                Integer sollicitudin, tortor a mattis commodo, velit urna rhoncus erat, vitae congue lectus dolor consequat libero. Donec leo ligula, maximus et pellentesque sed, gravida a metus. Cras ullamcorper a nunc ac porta. Aliquam ut aliquet lacus, quis faucibus libero. Quisque non semper leo.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
]
`;
